tests/project_tests.rs

```diff
@@ -47,3 +47,29 @@ async fn add_channel() {
     .unwrap();
     assert!(project.channels().contains(&local_channel));
 }
+
+#[tokio::test]
+async fn description() {
+    // Get a pixi instance
+    let pixi = PixiControl::new().unwrap();
+    pixi.init().await.unwrap();
+
+    let _command = "project description set \"Hello world\"";
+
+    // // Set the description
+    // let result = pixi
+    //     .run(run::Args {
+    //         task: command.split(" ").map(|s| s.to_string()).collect(),
+    //         ..Default::default()
+    //     })
+    //     .await
+    //     .unwrap();
+
+    // println!("{:?}", result);
+
+    // assert_eq!(result.exit_code, 0);
+    // assert_eq!(result.stdout.trim(), "Python 3.11.0");
+    // assert!(result.stderr.is_empty());
+
+    // let project = pixi.project().unwrap();
```

Comment:
I could not find an easy way to simple run `... project version get` from Rust + check the output + reload the toml and check everything is ok.

Did I miss it? Or should I try to add something like that in `common`?

Reply:
I could add a new struct/impl similar to `ProjectChannelAddBuilder` but given the number of new commands here, it appears to be tricky.

I can try to make something a bit less specific maybe.